api/prisma/migrations/20240214113648_link_to_version_post_script/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - Made the column `versionToId` on table `Links` required. This step will fail if there are existing NULL values in that column.
```

Comment:
Take it you can guarantee there are  no NULL values in the column?

Reply:
Yes, the script to run in between the migrations will mention if there was a problem filling all of these fields, so we'll know before running this migration

Reply:
I was messing with prisma migrations myself yesterday afternoon and discovered these warnings are auto generated. Apologies, I thought you wrote this